samples/nrf5340/netboot/src/main.c

```diff
@@ -41,6 +44,20 @@ int main(void)
 	bool valid = false;
 	uint8_t status = pcd_fw_copy_status_get();
 
+#ifdef CONFIG_PCD_LOCK_NETCORE_DEBUG
+	if (status == PCD_STATUS_LOCK_DEBUG) {
+		nrfx_nvmc_word_write((uint32_t)&NRF_UICR_NS->APPROTECT,
```

Comment:
Don't we need to sync after this to be sure that the data has been written?

Reply:
Excellent point, thank you.